graylog2-web-interface/src/views/components/datatable/Headers.tsx

```diff
@@ -43,17 +47,19 @@ const _headerField = (activeQuery: string, fields, field: string, prefix: (strin
 };
 
 const _headerFieldForValue = (activeQuery: string, field, value, span = 1, prefix = '') => (
-  <th key={`${prefix}${field}-${value}`} colSpan={span} className={styles.leftAligned}>
+  <CentratedTh key={`${prefix}${field}-${value}`} colSpan={span} className={styles.leftAligned}>
     <Value field={field} value={value} queryId={activeQuery} type={FieldType.Unknown}>{value}</Value>
-  </th>
+  </CentratedTh>
 );
 
 // eslint-disable-next-line jsx-a11y/control-has-associated-label
 const _spacer = (idx, span = 1) => <th colSpan={span} key={`spacer-${idx}`} className={styles.leftAligned} />;
 
 const columnPivotFieldsHeaders = (activeQuery: string, columnPivots: string[], actualColumnPivotValues: any[], series: Series[], offset = 1) => {
   return columnPivots.map((columnPivot, idx) => {
-    const actualValues = actualColumnPivotValues.map((key) => ({ path: key.slice(0, idx).join('-'), key: key[idx] || '', count: 1 }));
+    const actualValues = actualColumnPivotValues.map((key) => {
```

Comment:
I think this change can be removed. Using an arrow function without curly braces is perfectly fine.